### ISSUE TITLE
[Accessibility Audit] Missing heading levels in different pages

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/pressfreedomtracker.us#2003]
Few examples of pages as mentioned in the audit are:

* https://pressfreedomtracker.us/submit-incident/
* https://pressfreedomtracker.us/frequently-asked-questions/
* https://pressfreedomtracker.us/about/

The solution should be a mix of either adding `sr-only` headings of missing heading levels to define sections, or to adjust the heading levels to make them correct.

**Comment by ryanwhitney** [comment]
Seeing the same on https://pressfreedomtracker.us/all-incidents/ and individual incidents as well: https://pressfreedomtracker.us/all-incidents/photojournalist-injured-by-crowd-control-munition-at-new-jersey-protest/

both go h1->h3, though in /all-incidents the h3s are also within \<article\> tags.
